### PR TITLE
LPS-190582 add test to check if a draft does not schedule

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
Hi @adolfopa 

There is a problem with this PR?

File: modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/web/internal/scheduler/test/CheckKBArticleSchedulerJobConfigurationTest.java [hunk 1]
```diff
@@ -121,6 +122,40 @@ public void testExpireKBArticle() throws Exception {
 			WorkflowConstants.STATUS_EXPIRED, kbArticle.getStatus());
 	}
 
+	@Test
+	public void testNotPublishKBArticleIfKBArticleIsDraft() throws Exception {
```
Comment (adolfopa) [inline modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/web/internal/scheduler/test/CheckKBArticleSchedulerJobConfigurationTest.java:126]: Shouldn't this be "...DoNotPublish..."? (see `testDoNotExpireKBArticleIfKBArticleIsScheduled`, for example).

**Comment by adolfopa** [comment]
@AliciaGarciaGarcia Sorry for the delay :pray:

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/web/internal/scheduler/test/CheckKBArticleSchedulerJobConfigurationTest.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/knowledge-base/knowledge-base-test/src/testIntegration/java/com/liferay/knowledge/base/web/internal/scheduler/test/CheckKBArticleSchedulerJobConfigurationTest.java]: thanks!

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by SylviaLuan** [comment]
ci:test:relevant

**Comment by SylviaLuan** [comment]
ci:forward

**Pull request closed without merging**